packages/fiori/src/Timeline.ts

```diff
@@ -111,6 +190,112 @@ class Timeline extends UI5Element {
 			: Timeline.i18nBundle.getText(TIMELINE_ARIA_LABEL);
 	}
 
+	get showBusyIndicatorOverlay() {
+		return !this.growsWithButton && this.loading;
+	}
+
+	get growsOnScroll(): boolean {
+		return this.growing === TimeLineGrowingMode.Scroll;
+	}
+
+	get growingButtonIcon() {
+		return this.layout === TimelineLayout.Horizontal ? process : drillDown;
```

Comment:
The new best practice is to import all icons in the .tsx file along with the other dependencies, and have such a function there, but it's OK, don't change it now, just FYI for the future.